aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -416,7 +421,7 @@ Tensor eye(int64_t n,
     c10::optional<Device> device,
     c10::optional<bool> pin_memory) {
   // the default value of `m` equals to `n`
-  return native::eye(n, n, dtype, layout, device, pin_memory);
+  return at::_ops::eye_m::call(n, n, dtype, layout, device, pin_memory);
```

Comment:
What's this `at::_ops::eye_m` notation? First time I see it.

Reply:
Hmm, I thought we don't support unpacked TensorOptions at::eye, but looks like we do. This version is usually called from codegen, but it's codegen friendly API where none of the TensorOptions c++ api packing has happened. Calls like at::eye are internally implemented with it.